layouts/_default/summary.html

```diff
@@ -19,11 +19,14 @@ <h2 class="list__title post__title">
 		<div class="list__meta meta">{{ . }}</div>
 		{{- end }}
 	</header>
+	{{- if eq .Params.summary "" }}
+	{{- else }}
 	<div class="content list__excerpt post__content clearfix">
 		{{ .Summary }}
 	</div>
+	{{- end }}
 	{{- if .Site.Params.readmore }}
-	{{- if .Truncated }}
+	{{- if .Params.Truncated }}
```

Comment:
According to [the Hugo spec](https://gohugo.io/content-management/summaries/#example-first-10-articles-with-summaries), `.Truncated` seems to be the right variable.

What did you have in mind with this change?

Reply:
I stumbled across that control of the "Summaries" when testing. On trying it out, I realized that the truncation was global only, not on page level. But then I am new to Hugo and may have an issue with understanding the finer points of the internal mechanics, so I may be way off here.

In the final variant of the theme, I ended up introducing another page-level parameter that allowed to a) disable, or b) specify explicitly, or c) use the built-in mechanics for the "summary" (see fork and branch there, same license, of course).

Reply:
Well, accoridng to the docu and also my testing, this is a page variable already. So I'd change it back.